pastas/stressmodels.py

```diff
@@ -210,7 +221,7 @@ class StressModel(StressModelBase):
     ----------
     stress: pandas.Series
         pandas Series object containing the stress.
-    rfunc: rfunc class
+    rfunc: rfunc class or instance
```

Comment:
Already change description to demand an instance? Add deprecation warning to docstring here as well?